### ISSUE TITLE
Demo password don't works

### ISSUE THREAD

**Issue by dfuentes141** [opensourcepos/opensourcepos#2231]
I try to login on the demo with
 
 username: admin
 password: pointofsale

but don't work, sorry for my english

**Comment by jekkos** [comment]
Server was reset (to master branch for now, sorry @SteveIreland, but someone should validate my fix)

**Comment by odiea** [comment]
502 Bad Gateway
nginx/1.14.0 (Ubuntu)

**Comment by odiea** [comment]
Not accepting Password and Name. Invalid. Needs reset.

**Comment by SteveIreland** [comment]
I just pushed india-gst so demo was rebuilt with the india-gst flavor of OSPOS.  It is accepting the user id and password again.

**Comment by jekkos** [comment]
Maybe we also need to disable changing username as this could be the reaason behind disabling the default login

**Comment by odiea** [comment]
Something needs to be done with this.

**Comment by SteveIreland** [comment]
Good point.  Maybe in demo mode user number 1 cannot be changed at all?

**Comment by jekkos** [comment]
Well yes I would apply same logic as with passwords, no change if in testing mode

**Comment by daN4cat** [comment]
No evidence but I would bet a beer on the name behind this silly playing.

**Comment by WebShells** [comment]
Me sitting back watching all fingers pointing to @mmmmSteveIreland... Stop messing things 😑

**Comment by SteveIreland** [comment]
LOL

**Comment by odiea** [comment]
A Database Error Occurred
Unable to connect to your database server using the provided settings.

Filename: core/CodeIgniter.php

Line Number: 518

**Comment by daN4cat** [comment]
@jekkos you surely made people willing to embrace docker with this demo site... :-P

**Comment by SteveIreland** [comment]
I didn't do it.

**Comment by jekkos** [comment]
I think for a demo site it's the perfect solution, its sort of isolated from the real host and quick to redeploy or to upgrade a component (like bumping to php 7.2 mean just changing versions in the base container).
But still wonder how it got into this state. Maybe someone found a way to change db settings at runtime.

**Comment by jekkos** [comment]
no the mysql container just crashed

**Comment by odiea** [comment]
502 Bad Gateway
nginx/1.14.0 (Ubuntu)

**Comment by jekkos** [comment]
Ok yes indeed, something got stuck, there were quite some redeploys and then a process died inside the container, which didn't stop it completely for some reason. Had to restart the whole daemon to get it up again. Now @SteveIreland's branch is deployed there.

So I added some more logging to know next time what caused the issue.

**Comment by jekkos** [comment]
Also enabled the redis backend cache for weblate, seems to run a little smoother now

**Comment by odiea** [comment]
Does not have current 3.3.0. No attributes or Tax stuff.

**Comment by odiea** [comment]
Now it has the current version but the login is now displaying the following.
![2018-12-24 14_34_25-window](https://user-images.githubusercontent.com/10962177/50548884-5c0f6180-0c11-11e9-87d9-3d35811100b1.png)

**Comment by jekkos** [comment]
I think the problem appears when I we switch branches, then database is not always reinitialized.

**Comment by odiea** [comment]
A Database Error Occurred
Unable to connect to your database server using the provided settings.

Filename: core/CodeIgniter.php

Line Number: 518

**Comment by odiea** [comment]
Demo Server needs reset. All permissions but sales have been removed.

**Comment by jekkos** [comment]
Ok I have done a reset. Will need to add another check if this happens again.

**Comment by odiea** [comment]
Needs reset again.

**Comment by jekkos** [comment]
Ok it was reset. I guess we'll have to disable disabling permission for the employees module as well

**Comment by odiea** [comment]
502 Bad Gateway
nginx/1.14.0 (Ubuntu)
